### DISCUSSION TITLE
How to override Border radius?

### DISCUSSION THREAD

**Discussion by premdasvm** [element-plus/element-plus#6405] [Q&A]
Hi, how do I override the `el-dialog`'s border radius?

```vue
  <el-dialog
    :model-value="leadFormDialog"
    :title="title"
    center
    :style="{ borderRadius: `var(--el-border-radius-round)` }"
  >
  ```

**Comment by msidolphin** [comment]
I think we should add `--el-dialog-border-radius` variable

**Reply by premdasvm** [reply]
Can u show me an example, please?

**Reply by msidolphin** [reply]
> Can u show me an example, please?

<img width="1483" alt="image" src="https://user-images.githubusercontent.com/26672484/157582132-131eae1d-6561-496f-aa58-d06f0500b9de.png">

Now border-radius is a css variable, you can override it
